tools/sz_repo_cli/lib/src/common/src/concurrent_package_task_runner.dart

```diff
@@ -138,45 +142,48 @@ class _MaxConcurrentPackageStreamTransformer
 
 class _PackageTaskStatusUpdater {
   final Package package;
-  final DateTime Function() getCurrentDateTime;
-  final _controller = BehaviorSubject<PackageTaskStatus>();
-  Stream<PackageTaskStatus> get statusStream => _controller;
-  Running _running;
+  final DateTime Function()? getCurrentDateTime;
```

Comment:
This is not required but from what I can see (could be wrong) some methods would throw if this is actually null.